test.js

```diff
@@ -3,6 +3,12 @@
 const test = require('tape')
 const mewt = require('./lib')
 
+const createAssertMewt = t => input => {
+  t.equal(typeof input.$set, 'function')
+  t.equal(typeof input.$unset, 'function')
+  // TODO any other way to test? Cannot check if instanceof Proxy
```

Comment:
I think this is an adequate way to test if we are looking at a `mewt`, so no need for a TODO here.